examples/swap/cli/getsecret.js

```diff
@@ -1,5 +1,6 @@
 var ChainAbstractionLayer = require('../../../dist/index.cjs.js')
-const { Client, providers, networks, crypto } = ChainAbstractionLayer
+const { Client, providers, crypto } = ChainAbstractionLayer
+const networks = providers.bitcoin.networks
```

Comment:
The structure is a bit ugly atm. This would be nicer: 
`bitcoin.providers`
`bitcoin.networks`
a big restructure would be needed though, which is why i've not done this in this PR.